<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>image/Youtube.png</normaloff>image/Youtube.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>131</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한수원 한울림OTF</family>
    </font>
   </property>
   <property name="title">
    <string>음악 종류</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="classic">
      <property name="text">
       <string>클래식</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="jazz">
      <property name="text">
       <string>재즈</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="k_pop">
      <property name="text">
       <string>K POP</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>한국어</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>가사 없는 종류</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="bgm">
      <property name="text">
       <string>BGM</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QCheckBox" name="k_hiphop">
      <property name="text">
       <string>힙합</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QCheckBox" name="k_ballard">
      <property name="text">
       <string>발라드</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QCheckBox" name="k_trot">
      <property name="text">
       <string>트로트</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>195</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한수원 한울림OTF</family>
    </font>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한수원 한돋움OTF</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>오늘의 한마디</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한수원 한울림</family>
    </font>
   </property>
   <property name="text">
    <string>감정 선택</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_today">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>195</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한수원 한울림OTF</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_emotion">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한수원 한울림</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>감정 읽기</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>104</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한수원 한돋움OTF</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>오늘의 감정은?</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_feeling">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한수원 한울림</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_line">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>461</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(206, 19, 18);
border:0px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>601</width>
     <height>2</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(206, 19, 18);
border:0px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_title">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>451</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>한수원 한돋움OTF</family>
    </font>
   </property>
   <property name="text">
    <string> Title</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한수원 한돋움OTF</family>
    </font>
   </property>
   <property name="text">
    <string>▶ 지금 나오는 플레이리스트 :  </string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>265</y>
     <width>118</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한수원 한돋움OTF</family>
    </font>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_logout">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>113</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>한수원 한울림OTF</family>
    </font>
   </property>
   <property name="text">
    <string>로그아웃</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stop">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>370</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한수원 한돋움OTF</family>
    </font>
   </property>
   <property name="text">
    <string>play/stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_allstop">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>370</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한수원 한돋움OTF</family>
    </font>
   </property>
   <property name="text">
    <string>all_stop</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="lbl_feeling">
    <property name="font">
     <font>
      <family>한수원 한돋움OTF</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>기분 좋아 </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="text_1">
    <property name="font">
     <font>
      <family>한수원 한돋움OTF</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>보여요. </string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>7</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="lbl_name">
    <property name="font">
     <font>
      <family>한수원 한돋움OTF Bold</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="font">
     <font>
      <family>한수원 한돋움OTF</family>
     </font>
    </property>
    <property name="text">
     <string>님. 무슨 노래 듣고 싶으세요?</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
